redash/models/__init__.py

```diff
@@ -122,6 +123,7 @@ def to_dict(self, all=False, with_permissions_for=None):
             "syntax": self.query_runner.syntax,
             "paused": self.paused,
             "pause_reason": self.pause_reason,
+            "apply_auto_limit": self.query_runner.supports_auto_limit
```

Comment:
```suggestion
            "supports_auto_limit": self.query_runner.supports_auto_limit
```

This should be `supports_auto_limit` too. 

Sorry for the nits, it just that these things are very hard to change later.